data/model_weights/hifigan/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Jungil Kong
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```

Comment:
I think this file is added here by mistake?

Reply:
I have copied this file to where the content should exist. This was part of the `FastSpeech2` directory (https://github.com/ARBML/klaam/tree/main/FastSpeech2/hifigan).

As we restructured the setup of the project, `FastSpeech2` contained a lot of hard-coded relative paths to `FastSpeech2` project, which isn't a good practice and won't allow the current setup to work unless we do some dirty hacks which could cause more problems in later developments of the project.

Therefore, I needed to move `configs` and `weights` files to be outside of the logic code. To follow the license information, I have copied the `LICENSE` file where I have moved the `configs` and `weights` files.

This can be removed if you think it isn't necessary.

Reply:
Before investing anything related to FastSpeech, I think we should remove it @zaidalyafeai please comment on this.